test/specs/scenarios/accessibility.js

```diff
@@ -0,0 +1,186 @@
+      documentUpload,
+      crossDeviceIntro,
+      livenessIntro,
+      poaDocumentSelection,
+      poaIntro,
+      basePage
+    } = pageObjects
+
+    const copy = basePage.copy(lang)
+
+    const goToPoADocumentSelectionScreen = async () => {
+      driver.get(localhostUrl + `?poa=true&async=false&useWebcam=false`)
+      welcome.primaryBtn.click()
+      poaIntro.clickStartVerificationButton()
+    }
+
+    const goToCrossDeviceScreen = async () => {
+      welcome.primaryBtn.click()
+      documentSelector.passportIcon.click()
+      documentUpload.crossDeviceIcon.click()
+      crossDeviceIntro.continueButton.click()
+    }
+    
+    //Welcome
+    it('should verify accessibility for the welcome screen', async () => {
+      driver.get(localhostUrl + `?language=${lang}`)
+      runAccessibilityTest(driver)
+    })
+
+    it('should verify focus management for the welcome screen', async () => {
+      driver.get(localhostUrl + `?language=${lang}`)
+      welcome.verifyFocusManagement()
+    })
+
+    //Cross Device Sync
+    it('should verify accessibility for the cross device intro screen', async () => {
+      driver.get(localhostUrl + `?language=${lang}`)
+      welcome.primaryBtn.click()
+      documentSelector.passportIcon.click()
+      documentUpload.crossDeviceIcon.click()
+      runAccessibilityTest(driver)
+    })
+
+    it('should verify accessibility for the cross device screen', async () => {
+      driver.get(localhostUrl + `?language=${lang}`)
+      goToCrossDeviceScreen()
+      runAccessibilityTest(driver)
+    })
+
+    // Document Selector
+    it('should verify accessibility for the document selector screen', async () => {
+      driver.get(localhostUrl + `?language=${lang}`)
+      welcome.primaryBtn.click()
+      runAccessibilityTest(driver)
+    })
+
+    //Document Upload
+    it('should verify accessibility for the uploader screen', async () => {
+      goToPassportUploadScreen(driver, welcome, documentSelector, `?language=${lang}`)
+      runAccessibilityTest(driver)
+    })
+
+    //FIXME: This is commented out due to the accessibility violations we need to fix.
+    // it('should verify accessibility for the document upload confirmation screen', async () => {
+    //   goToPassportUploadScreen(driver, welcome, documentSelector, `?language=${lang}`)
+  
+    //   documentUpload.getUploadInput()
+    //   documentUpload.upload('passport.jpg')
+    //   runAccessibilityTest(driver)
+    // })
+
+    //Face
+    it('should verify accessibility for the take a selfie screen', async () => {
+      goToPassportUploadScreen(driver, welcome, documentSelector,`?language=${lang}&async=false`)
+      uploadFileAndClickConfirmButton(documentUpload, confirm, 'passport.jpg')
+      runAccessibilityTest(driver)
+    })
+  
+    //FIXME: This is commented out due to the accessibility violations we need to fix.
+    // it('should verify accessibility for the selfie confirmation screen', async () => {
```

Comment:
Which rule is failing here?

Reply:
color-contrast
```AssertionError: 1 violations found:
- Rule ID: color-contrast
  Elements must have sufficient color contrast (1 elements affected)
  Help: https://dequeuniversity.com/rules/axe/3.3/color-contrast?application=webdriverjs
: expected 1 to equal 0
```

Reply:
Minor: probably worth just adding to the FIXME comment that it's the `color-contrast` rule that's failing since that's a less immediately obvious accessibility issue